Directory.Build.props

```diff
@@ -16,6 +16,7 @@
     <!-- Configure Sentry CLI -->
     <SentryOrg>sentry-sdks</SentryOrg>
     <SentryProject>sentry-dotnet</SentryProject>
+    <SentryUploadSources>true</SentryUploadSources>
```

Comment:
One concern I have is that if users pick `<EmbedAllSources>true</EmbedAllSources>`, we'll also have sources and display them. Even if `SentryUploadSources=false`.

So on the docs we definitely need a _<Note> If you're using EMbedAllSources` Sentry will already have the sources through uploading the PDBs_

Reply:
Already in the docs PR.
https://github.com/getsentry/sentry-docs/pull/6183

Reply:
... but I could also check for it here to be safe. Will do.

Reply:
Done.